pkg/meta/tkv.go

```diff
@@ -2304,7 +2304,49 @@ func (m *kvMeta) scanDeletedSlices(ctx Context, scan deletedSliceScan) error {
 	return nil
 }
 
-func (m *kvMeta) scanDeletedFiles(ctx Context, scan deletedFileScan) error {
+func (m *kvMeta) scanPendingSlices(ctx Context, scan pendingSliceScan) error {
+	if scan == nil {
+		return nil
+	}
+
+	// slice refs: Kiiiiiiiissss
+	klen := 1 + 8 + 4
+	keys, err := m.scanKeys(m.fmtKey("K"))
```

Comment:
use scan() to fetch values, faster than `incrby(0)` for every key

Reply:
Done